buildvertical.targets.save

```diff
@@ -28,7 +28,8 @@
           Outputs="fake"
           Returns="@(Project)"
           DependsOnTargets="TrimUnconvertedProjects"
```

Comment:
I plan to delete this once we determine that we don't need any dynamic configuring.